docs/sources/plugins/developing/dataframe.md

```diff
@@ -45,15 +45,15 @@ With dataframes, each column is represented by a **Field**. So the essence of a
 
 One restriction on dataframes is that all Fields in the frame must be of the same length to be a valid dataframe.
```

Comment:
"Field" in this context is not a proper noun. Please make it lowercase wherever applicable.

Reply:
`Field` is the name of a field in the struct. https://github.com/grafana/grafana-plugin-sdk-go/blob/master/data/frame.go#L35 Whats a good way to indicate the connection between docs and something in the code?